src/CSnakes.SourceGeneration/CaseHelper.cs

```diff
@@ -1,16 +1,16 @@
-﻿namespace CSnakes
-{
-    public static class CaseHelper
-    {
-        public static string ToPascalCase(this string snakeCase)
-        {
-            return string.Join("", snakeCase.Split('_').Select(s => s.Length > 1 ? char.ToUpperInvariant(s[0]) + s.Substring(1) : "_"));
-        }
-
-        public static string ToLowerPascalCase(this string snakeCase)
-        {
-            // Make sure the first letter is lowercase
-            return char.ToLowerInvariant(snakeCase[0]) + ToPascalCase(snakeCase).Substring(1);
-        }
-    }
-}
+﻿namespace CSnakes
+{
+    public static class CaseHelper
+    {
+        public static string ToPascalCase(this string snakeCase)
+        {
+            return string.Join("", snakeCase.Split('_').Select(s => s.Length > 1 ? char.ToUpperInvariant(s[0]) + s.Substring(1) : "_"));
+        }
+
+        public static string ToLowerPascalCase(this string snakeCase)
+        {
+            // Make sure the first letter is lowercase
+            return char.ToLowerInvariant(snakeCase[0]) + ToPascalCase(snakeCase).Substring(1);
+        }
+    }
+}
```

Comment:
```suggestion
﻿namespace CSnakes;
public static class CaseHelper
{
    public static string ToPascalCase(this string snakeCase)
    {
        return string.Join("", snakeCase.Split('_').Select(s => s.Length > 1 ? char.ToUpperInvariant(s[0]) + s.Substring(1) : "_"));
    }

    public static string ToLowerPascalCase(this string snakeCase)
    {
        // Make sure the first letter is lowercase
        return char.ToLowerInvariant(snakeCase[0]) + ToPascalCase(snakeCase).Substring(1);
    }
}
```

Reply:
Using this as an opportunity to fix the namespace to be file-scoped

Reply:
This would be a diff-breaking change.
I suggest performing this change as a separate PR **with no other big PR in flight**.